tensorflow/contrib/cudnn_rnn/python/ops/cudnn_rnn_ops.py

```diff
@@ -959,6 +959,7 @@ def _cudnn_rnn(inputs,
                direction=CUDNN_RNN_UNIDIRECTION,
                dropout=0.,
                seed=0,
+               sequence_lengths=None,
```

Comment:
I prefer to rename `sequence_lengths` to `input_lengths` and move it above `input_mode` (as the first default arg). It's an internal func, shouldn't cause issue.

Reply:
I am a little confused here. Do you mean all the internal funcs (with prefix `_`) will use `input_lengths` and all the others still use `sequence_lengths`?